_posts/2020-10-27-a-closer-look-at-the-interpreter.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "A closer look at the interpreter"
+tags: BEAM erts jit
+author: John Högberg
+---
+
+In my previous post we had a look at BEAM, and now that we're more familiar
```

Comment:
put a link to the previous post if possible

Reply:
Of course. :)

I'm thinking of merging this post with the [prior one](#17), making one long post rather than a series of 3-4, but I can't decide one way or the other. Do you think that would fit better?

Reply:
I personally like to write very long posts, but the sort of downside with it is that people open them up in a tab and then never reads them again. Getting stuff in shorter digestible bites makes sense. In this case I could mostly get everything I think made sense there (how the interpreter is kind of framed, the limitations it runs into) and it lays ground towards wanting to see the JIT post that I now assume is incoming.

Linking across posts is probably a good format. If I had to compare them to TV/Movies:
- I like to do very long posts and I guess that gets into the "Lord of the Rings director cut" in terms of duration and you lose a ton of people just by looking at the size of things, but you're allowing yourself to explore all the tangents you feel are worth poking at (I'm trying to shorten my own stuff because it's too easy to lose people outside of books or talks)
- movie-sized larger posts (what you'd get here with a merge)
- shorter-sized posts that go like episodes of a TV show. You even got the cliffhanger there and I'm asking for "last week on OTP team's blog posts" reminders that make it easier to take em slow if you don't binge

For the last two, the thing I'd make a deciding factor is whether what you aim for is having a self-standing article that's easy to link around ("here you go, this explains whatever you need all in one place"), or whether you're looking to establish a cadence and habit ("we make a technical post every x weeks, so stay tuned"). The former works well for growing a reference set, but the latter for engaging your audience over time.

Reply:
Thanks, I think I'll keep the episodic format.